lib/tasks/fake_data.rake

```diff
@@ -18,18 +18,21 @@ if Rails.env.development?
       division = FactoryBot.create(:division)
 
       # Create admin user
-      user = FactoryBot.create(:user, :admin,
-        email: "admin@example.com",
-        password: "xxxxxxxx",
-        password_confirmation: "xxxxxxxx"
-      )
-      user.add_role :admin, Division.root
+      user = User.find_by(email: "admin@example.com")
+      if user.empty?
```

Comment:
When I try to run `rake dev:fake_data`, I get `NoMethodError: undefined method `empty?' for nil:NilClass` for this line.

I found replacing this with `user.nil?` does work.